spec.html

```diff
@@ -207,7 +277,7 @@ <h1>AsyncIterator ( )</h1>
         <h1>Value Properties of the AsyncIterator Constructor</h1>
         <emu-clause id="sec-asynciterator.prototype">
           <h1>AsyncIterator.prototype</h1>
-          <p>The initial value of AsyncIterator.prototype is %AsyncIterator.prototype%.</p>
+          <p>The initial value of AsyncIterator.prototype is %AsyncIteratorPrototype%.</p>
```

Comment:
please revert this

Reply:
why? the dot form is always preferred, editorially.

Reply:
@ljharb right, i'm asking that it be reverted back to using the dot form.

Reply:
This refers to the [`%AsyncIteratorPrototype%`](https://tc39.es/ecma262/#sec-asynciteratorprototype) intrinsic specified in <https://tc39.es/ecma262/#sec-well-known-intrinsic-objects>.